app/models/iam_user_identity.rb

```diff
@@ -52,4 +53,14 @@ def set_expire
   def uuid
     Digest::UUID.uuid_v5(@iam_sec_id, @iam_icn)
   end
+
+  # Return a single mhv id from a possible comma-separated list value attribute
+  def self.valid_mhv_id(id_from_profile)
+    # TODO: site login fails if multiple _different_ MHV IDs are present
+    # Should this code be doing the same? Or at least logging a warning?
```

Comment:
Some questions about this.

-  What kind of issues does this cause, would their messages potentially diverge between accounts? 
- How does a veteran resolve having multiple MHV ids?

If the issues doesn't cause data issues, I think logging would be sufficient.

Reply:
Yes, they'd potentially have different message contents between the two accounts.

MHV website  and VA.gov (as of the SSOe integration) prevent a user in such state from signing  in,  and presents a  message instructing them to call the help desk to resolve.

Reply:
👍 ok, sounds like we should do the same thing as .gov and raise an error (and map that error to a helpful in-app message)